src/base/utils/string.h

```diff
@@ -60,7 +60,7 @@ namespace Utils
         {
             if (str.length() < 2) return str;
 
-            for (auto const quote : quotes) {
+            for (auto const &quote : quotes) {
```

Comment:
adding & adds another indirection which is not necessary here, also you should switch `auto const` -> `const auto`

Reply:
I had a warning here:
`warning: loop variable 'quote' of type 'const QChar' creates a copy from type 'const QChar'
use reference type 'const QChar &' to prevent copying`

Reply:
ok, just switch the order of "auto" & "const".